pkg/controllers/installer/helpers_test.go

```
@@ -298,6 +300,74 @@ func addRevisionAndWaitForSuccess(ctx context.Context, providerNames ...string)
+
+	By("Adding revision to ClusterAPI status", func() {
+		clusterAPI.Status.Revisions = append(clusterAPI.Status.Revisions, apiRev)
+		clusterAPI.Status.DesiredRevision = apiRev.Name
+		Expect(cl.Status().Update(ctx, clusterAPI)).To(Succeed())
+	})
+
+	return apiRev
+}
+
+// setCompatibilityRequirementConditions sets Admitted and Compatible conditions on a CompatibilityRequirement.
+func setCompatibilityRequirementConditions(ctx context.Context, name string, admitted, compatible bool) {
```

golangci-lint (2.12.2)

[error] setCompatibilityRequirementConditions - admitted always receives true

(unparam)